components/atoms/Checkbox/checkbox.tsx

```diff
@@ -29,7 +29,7 @@ const Checkbox = React.forwardRef<React.ElementRef<typeof CheckboxPrimitive.Root
         {label && (
           <label
             htmlFor={getId()}
-            className="ml-3 text-sm font-medium leading-none cursor-pointer text-light-slate-12 peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
```

Comment:
All the `font-medium` classes related to h1-h6 have been removed in all files as the base style for h1-h6 handles this now.